common/kubernetes/configs.go

```diff
@@ -92,7 +86,7 @@ func GetKubeClientConfig(uri *url.URL) (*kube_rest.Config, error) {
 		if configOverrides.ClusterInfo.Server != "" {
 			kubeConfig.Host = configOverrides.ClusterInfo.Server
 		}
-		kubeConfig.GroupVersion = &schema.GroupVersion{Version: configOverrides.ClusterInfo.APIVersion}
+		kubeConfig.GroupVersion = &schema.GroupVersion{Version: APIVersion}
```

Comment:
this should just be `"k8s.io/api/core/v1".SchemeGroupVersion`

Reply:
It should be `"k8s.io/apimachinery/pkg/apis/meta/v1". SchemeGroupVersion`.

Done